lib/ansible/plugins/connection/docker.py

```diff
@@ -132,8 +132,8 @@ def _get_docker_version(self):
 
         cmd, cmd_output, err, returncode = self._old_docker_version()
         if returncode == 0:
-            for line in cmd_output.split('\n'):
-                if line.startswith('Server version:'):  # old docker versions
+            for line in cmd_output.split(b'\n'):
+                if line.startswith(b'Server version:'):  # old docker versions
```

Comment:
I "slept on it", and I think I agree.  It do think it make sense to convert it in one place.  I'll look into it and test.